### PR TITLE
Add popup for databrowser

### LINKED ISSUE #590: Include solid-auth-client popup in node-solid-server

### PR DESCRIPTION
**Pull request by RubenVerborgh**
Closes #590.

### CODE REVIEW

**Comment by RubenVerborgh** [comment]
This depends on the publication of mashlib with solid-auth-client 1.0.0 (through solid-ui), which @dan-f is working on. Popup will need a regeneration for minor issues.

I did not find how to trigger the generated popup HTML from mashlib, so any help would be appreciated.

**Comment by dan-f** [comment]
@RubenVerborgh - I can show you how to trigger the login. Easier to explain in person than over GH comments.

**Comment by RubenVerborgh** [comment]
@dan-f @timbl This one's ready for testing now.

**Comment by dan-f** [comment]
Popup needs to be templated with the name of the app (Solid Home?)

File: static/databrowser.html [hunk 1]
```diff
@@ -2,23 +2,27 @@
...
-
-      var subject = UI.rdf.namedNode(uri)
-      UI.outline.GotoSubject(subject, true, undefined, true, undefined)
-    })
-})
-</script>
+        UI.authn.checkUser()
+          .then(function () {
+            var uri = window.location.href
+            var dataUri = window.document.title = uri.slice(0, uri.lastIndexOf('/')+1) + ''
```
Comment (tomasklapka) [inline static/databrowser.html]: this uri slicing is causing #603

**Pull request merged**